pkg/config/nodetreemodel/config.go

```diff
@@ -51,6 +51,18 @@ type ntmConfig struct {
 	// Bellow are all the different configuration layers. Each layers represents a source for our configuration.
 	// They are merge into the 'root' tree following order of importance (see pkg/model/viper.go:sourcesPriority).
 
+	// schema holds all the settings with or without value. Settings are added to the schema through BindEnv and
```

Comment:
Is this table an accurate summation?

```
           +---------+------------+
           | BindEnv | SetDefault |
           +---------+------------+
AllKeys    | yes     | yes        |
AlSettings | -       | yes        |
Get        | *       | yes        |
schema     | yes     | yes        |
defaults   | -       | yes        |

*: Yes, but if and only if an env var is set with a value
```

Reply:
I think:

```
           +---------+------------+
           | BindEnv | SetDefault |
           +---------+------------+
AllKeys    | yes     | yes        |
AlSettings | *       | yes        |
Get        | *       | yes        |
schema     | yes     | yes        |
defaults   | -       | yes        |

*: Yes, but if and only if an env var is set with a value
```

Reply:
Right, that makes sense. Ok, I understand the difference now.